packages/generator/src/parser/parser-types-common.ts

```diff
@@ -163,3 +120,14 @@ export interface SwaggerPathParameter extends SwaggerDescribed {
 export interface SwaggerDescribed {
   description?: string;
 }
+
+export interface JoinedEntityMetadata {
```

Comment:
Minor: I find this name not so good. `SetWithType` perhaps? The swagger definition is perhaps part of the entitySet?

Reply:
This has been here forever, I just moved it from one file to another. Not sure if this was exported before, though. Anyways, I don't love the name either, but I still like it better than `SetWithType`. I couldn't come up with anything substantially better and at least this one wins with "Gewohnheitsrecht" ;)

Reply:
JoinedData could be anything. It does not tell you what it is. But let's keep it since refactoring a name is easy also later. Which means never :-).